platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -993,9 +986,21 @@ private EventLinker buildEventLinker(@NonNull final List<Predicate<EventDescript
      * Build the preconsensus event file manager.
      */
     @NonNull
-    private PreconsensusEventFileManager buildPreconsensusEventFileManager() {
+    private PreconsensusEventFileManager buildPreconsensusEventFileManager(
+            final EmergencyRecoveryManager emergencyRecoveryManager) {
         try {
-            return new PreconsensusEventFileManager(platformContext, OSTime.getInstance(), recycleBin, selfId);
+            final PreconsensusEventFileManager manager =
+                    new PreconsensusEventFileManager(platformContext, OSTime.getInstance(), recycleBin, selfId);
+
+            if (emergencyRecoveryManager.isEmergencyRecoveryFilePresent()) {
+                logger.info(
+                        STARTUP.getMarker(),
+                        "This node was started in emergency recovery mode, "
```

Comment:
Nit: unnecessary line break in string

Reply:
Technically, yes. But it bothers my OCD when the intellij hints push something over the line count boundary.

![Screenshot 2023-06-15 at 1 01 45 PM](https://github.com/hashgraph/hedera-services/assets/56973212/42800001-acdc-4f5c-85fe-a36779ea10a2)